code/game/objects/items/devices/personal_data_transmitter.dm

```diff
@@ -145,6 +145,7 @@
 	name = "\improper PDT bracelet"
 	desc = "A personal data transmitter bracelet, also known as a PDT, is a form of personal locator typically surgically implanted into the body of extrasolar colonists, among others. Its purpose is to allow rapid location of the associated personnel anywhere within a certain radius of the receiving equipment, sometimes up to 30km distance. This bracelet forms part of the PDT/L variant, which is a wearable version of the PDT technology. Both it and the linked locator tube share a serial number for ease of detection in case of mixup."
 	icon_state = "pdt_watch"
```

Comment:
remove the icon state definition here, in 

/obj/item/clothing/accessory/pdt_bracelet/Initialize()

change the [icon_state]

to [dummy_icon_state]